Riot/AppDelegate.m

```diff
@@ -4232,11 +4237,26 @@ - (void)askNativeWidgetPermissionWithWidget:(Widget*)widget completion:(void (^)
 
 #pragma mark - Call status handling
 
+/// Returns a suitable height for call status bar. Considers safe area insets if available and notch status.
+- (CGFloat)calculateCallStatusBarHeight
+{
+    CGFloat result = CALL_STATUS_BAR_HEIGHT;
+    if (@available(iOS 11.0, *))
+    {
+        if (UIDevice.currentDevice.hasNotch)
+        {
+            //  this device has a notch (iPhone X +)
+            result += UIApplication.sharedApplication.keyWindow.safeAreaInsets.top;
+        }
```

Comment:
Instead of using hasNotch we could use this:
```
        static CGFloat kCallStatusBarSafeAreaExtraHeight = 24;        
        result = MAX(UIApplication.sharedApplication.keyWindow.safeAreaInsets.top + kCallStatusBarSafeAreaExtraHeight, CALL_STATUS_BAR_HEIGHT);
```